dashboard/src/components/AppView/AppControls/UpgradeButton.test.tsx

```diff
@@ -17,7 +18,7 @@ it("renders a redirect when clicking upgrade", () => {
 
   button.simulate("click");
   expect(push.mock.calls.length).toBe(1);
-  expect(push.mock.calls[0]).toEqual(["/ns/default/apps/foo/upgrade"]);
+  expect(push.mock.calls[0]).toEqual([url.app.apps.upgrade("foo", "default")]);
 });
 
 context("when a new version is available", () => {
```

Comment:
BTW, just noticed that the "Delete" button redirects to an invalid link (AppControls.tsx:86) generating a 404

Reply:
Fixed in #1809 (also audited for other hard-coded links there). Thanks.